wp_api/src/login.rs

```diff
@@ -98,6 +98,36 @@ pub enum OAuthResponseUrlError {
     UnsuccessfulLogin,
 }
 
+#[uniffi::export]
+pub fn create_application_password_authentication_url(
+    login_url: Arc<ParsedUrl>,
+    app_name: String,
+    app_id: Option<String>,
+    success_url: Option<String>,
+    reject_url: Option<String>,
```

Comment:
I wonder if it's worth linking to https://make.wordpress.org/core/2020/11/05/application-passwords-integration-guide/ in the docs?

Reply:
Added in https://github.com/Automattic/wordpress-rs/pull/250/commits/e6b5a05f685fd84977a7250354bb634f0284123f